core/pkg/ingress/status/status.go

```diff
@@ -19,21 +19,25 @@ package status
 import (
 	"fmt"
 	"net"
+	"os"
 	"sort"
 	"sync"
 	"time"
 
 	"github.com/golang/glog"
 
+	api "k8s.io/api/core/v1"
```

Comment:
call this `v1` or `corev1`. It gets pretty confusing when reading the code in these packages. For example, see https://github.com/kubernetes/ingress/pull/994/files#diff-636822cdb1e125937cd0afa244e18161R33.